docs/configuration.md

```diff
@@ -189,6 +194,15 @@ matchers:
 mute_time_intervals:
   [ - <string> ...]
 
+# Times when the route should be active. These must match the name of a
```

Comment:
I think it's worth mentioning here that an empty values means that the route is always active.

Reply:
Right, I will include this

Reply:
I update the doc.

Reply:
It is updated